roles/install-k8s/tasks/main.yml

```diff
@@ -65,7 +65,7 @@
   command: "kubectl taint nodes --all node-role.kubernetes.io/master-"
   environment:
     KUBECONFIG: /etc/kubernetes/admin.conf
-  when: node_type == "master"
+  when: node_type == "master" and workers_count == 0
```

Comment:
no need for an additional variable here, wondering if we can somehow add a logic to find the worker host counts.

Reply:
this may be helpful - https://www.reddit.com/r/ansible/comments/jj47g5/how_to_get_the_number_of_hosts_in_a_group/

Reply:
This is actually an existing variable that comes from the ansible json string created by kubetest2 if that's what you're requesting for. This is the variable that kubetest2 reads from the command we enter in the prow job.
Refer - https://github.com/ppc64le-cloud/kubetest2-plugins/blob/6cfa72a42957688d35e90a1da45cc853ec764ed0/data/data/config.tf#L23

Reply:
Something like this you mean - 
```
- name: remove taint
  command: "kubectl taint nodes --all node-role.kubernetes.io/master-"
  environment:
    KUBECONFIG: /etc/kubernetes/admin.conf
  vars:
    total_workers: "{{ groups['workers'] | length }}"
  when: node_type == "master" and total_workers == 0
```

Reply:
I echo with @mkumatag as this idea of adding condition based on `workers_count` variable might not leave the taint when we run ansible-playbook alone with workers in `hosts.yml` file
I mean independent of kubetest2.

Reply:
OR

```
- name: remove taint
  command: "kubectl taint nodes --all node-role.kubernetes.io/master-"
  environment:
    KUBECONFIG: /etc/kubernetes/admin.conf
  when: node_type == "master" and "{{ groups['workers'] | length }}" == 0
```

Reply:
> - name: remove taint
>   command: "kubectl taint nodes --all node-role.kubernetes.io/master-"
>   environment:
>     KUBECONFIG: /etc/kubernetes/admin.conf
>   when: node_type == "master" and "{{ groups['workers'] | length }}" == 0

I like this.

Reply:
okay. I'll test it once. and confirm back.